docs/install/cli.md

```diff
@@ -189,6 +189,10 @@ ktlint installGitPrePushHook
 
 `--relative`: Print files relative to the working directory (e.g. dir/file.kt instead of /home/user/project/dir/file.kt)
 
+`--patterns-from-stdin[=<delimiter>]`: Reads additional patterns from `stdin`, where the patterns are separated by `<delimiter>`. If `=<delimiter>` is omitted, newline is used as fallback delimiter. If an empty string is given, the `NUL` byte is used as delimiter instead.  
+`-z` is a shorthand for `--patterns-from-stdin=''`  
```

Comment:
In the PR description it is documented that the option is primarily added for usage in the git hooks. In what other use cases would this flag be useful?

As I do not expect this flag to be used in a lot of other situations, I would like that option `-z` is removed. In the git hooks I would prefer to used the longer more descriptive `--patterns-from-stdin=''` anyways. The additional complexity of the mutual exclusivity will not be needed then.

Reply:
> The additional complexity of the mutual exclusivity will not be needed then.

Sadly it is still needed, the mutual exclusivity has nothing to do with the `-z` shorthand, it exists because both the `--stdin` and `--patterns-from-stdin` options would try to read the entirety of the standard input, so only one of them can be given at the same time.

Reply:
Ah yes, clear.